tidb-cloud/serverless-limitations.md

```diff
@@ -50,7 +50,6 @@ We are constantly filling in the feature gaps between TiDB Serverless and TiDB D
 
 ### Others
 
-- [Time to live (TTL)](/time-to-live.md) is currently unavailable.
```

Comment:
Please remove the following line: https://github.com/pingcap/docs/blob/e74ac630da50844aaa3c60946ed26dc464269d4d/time-to-live.md?plain=1#L261

Reply:
done